CHANGELOG.md

```diff
@@ -99,7 +100,7 @@ Decidim::Notification.where(event_class: "Decidim::Proposals::ProposalEndorsedEv
 
 - **decidim-proposals**: Fix threshold_per_proposal method positive? for nil:NilClass when threshold is null or not defined. [\#3219](https://github.com/decidim/decidim/pull/3219)
 - **decidim-proposals**: Fix when I create a proposal I see the draft proposal from someone else! [\#3170](https://github.com/decidim/decidim/pull/3083)
- - **decidim-proposals**: Fix view hooks returning proposals that should not be shown [\#3175](https://github.com/decidim/decidim/pull/3175)
+- **decidim-proposals**: Fix view hooks returning proposals that should not be shown [\#3199](https://github.com/decidim/decidim/pull/3199)
```

Comment:
CHANGELOG changes should go to the `Fixed` section and should not modify previous versions changes

Reply:
This will get fixed with the next rebase